include/dca/phys/dca_step/cluster_solver/stdthread_qmci/stdthread_qmci_accumulator.hpp

```diff
@@ -27,13 +27,12 @@ namespace stdthreadqmci {
 
 template <class qmci_accumulator_type>
 class stdthread_qmci_accumulator : protected qmci_accumulator_type {
-  typedef typename qmci_accumulator_type::my_parameters_type parameters_type;
+  using ThisType = stdthread_qmci_accumulator<qmci_accumulator_type>;
+  using Parameters = typename qmci_accumulator_type::my_parameters_type;
   using Data = typename qmci_accumulator_type::DataType;
 
-  typedef stdthread_qmci_accumulator<qmci_accumulator_type> this_type;
-
 public:
-  stdthread_qmci_accumulator(parameters_type& parameters_ref, Data& data_ref, int id);
+  stdthread_qmci_accumulator(Parameters& parameters_ref, Data& data_ref, int meas_t_do, int id);
```

Comment:
The need to track the measurements here will disappear if we merge https://github.com/CompFUSE/DCA-develop/pull/191.